Dockerfile

```diff
@@ -80,9 +80,13 @@ ENV PATH="/usr/local/bundle/bin:${PATH}"
 
 RUN whoami
 
-# Download VA Certs
-COPY ./import-va-certs.sh .
-RUN ./import-va-certs.sh
+# Clone platform-va-ca-certificate and copy certs
+WORKDIR /srv
+RUN git clone --depth 1 https://github.com/department-of-veterans-affairs/platform-va-ca-certificate && \
+    cp platform-va-ca-certificate/VA*.cer /tmp && \
+    /bin/bash platform-va-ca-certificate/debian-ubuntu/install-certs.sh
```

Comment:
Should we remove the repo when we're done here?

Reply:
oh, sure. The intent was for this to be done in an earlier throwaway stage in my other dockerfile... Perhaps I can move this to the builder stage for this case? Nothing needs to be kept (including /tmp), right?

Reply:
JK moved it into the base instead to simplify permissions a bit. Ultimately it will probably land in an earlier stage, so this should be good for now